From 1e7c8bc24fe16c4ff77090d71d1398a04379106f Mon Sep 17 00:00:00 2001
From: Jun Miao <jun.miao@windriver.com>
Date: Fri, 13 Jul 2018 00:26:38 -0400
Subject: [PATCH] systemd:mount /var/log/jouranl in journald service

When reboot,system will print log "Faild unmounting /var",we should add a
function:mount the /var when systemd-journald shutdown.So that after journalctl
shutdown,unmount /var/log/journal/

Signed-off-by: Jun Miao <jun.miao@windriver.com>
---
 units/systemd-journald.service.in | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/units/systemd-journald.service.in b/units/systemd-journald.service.in
index 66b7c6a..983aa06 100644
--- a/units/systemd-journald.service.in
+++ b/units/systemd-journald.service.in
@@ -13,6 +13,8 @@ Requires=systemd-journald.socket
 After=systemd-journald.socket systemd-journald-dev-log.socket systemd-journald-audit.socket syslog.socket
 Before=sysinit.target
 
+RequiresMountsFor=/var/log/journal
+
 [Service]
 Type=notify
 Sockets=systemd-journald.socket systemd-journald-dev-log.socket systemd-journald-audit.socket
-- 
2.7.4

